shogun-config/src/main/java/de/terrestris/shogun/properties/UploadProperties.java

```diff
@@ -33,4 +32,8 @@
     @NestedConfigurationProperty
     private ImageFileUploadProperties image;
 
+    private String path;
```

Comment:
as `path` is a reserved keyword in PostgreSQL, I would prefer to rename this variable.

Reply:
nice, Röö, goiod catch, even though it would have been escaped properly, this can be the source of oddities. Cool.